mne_bids/io.py

```diff
@@ -105,9 +105,22 @@ def read_raw_bids(bids_fname, bids_root, return_events=True,
     bids_root : str
         Path to root of the BIDS folder
     return_events: bool
```

Comment:
should add `, optional` I think

Reply:
well all keyword arguments are optional. The function signature will tell you if it's optional or not.

Reply:
fair enough, I just thought the spec for the numpy docstring said to have the `optional` tag in it if they were kwargs

Reply:
ah indeed, I learned something today :)

Reply:
I can confirm, see [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) and search for `, optional`